cli/core/pkg/cli/catalog_test.go

```diff
@@ -106,7 +107,7 @@ func testHasUpdate(t *testing.T, multi *MultiRepo, numPluginsDowngraded int) {
 		}
 	}
 	require.Equal(t, numPluginsRequiringUpdate, numPluginsDowngraded)
-}
```

Comment:
Not sure I understand why this is commented out instead of removed.

Reply:
The reason I commented instead of removing the code was, as part of TKG-13912, we may remove or move the code in which we might use these test cases if we don't use then we will remove it as part of TKG-13912. 
@anujc25 let me know if we can remove it right away!

Reply:
I have discussed with Anuj, added `nolint's`, and un-commented the code.